lighthouse-core/report/report-generator.js

```diff
@@ -34,6 +34,23 @@ class ReportGenerator {
       return Math.round(totalScore * 100);
     };
 
+    const getItemRating = (value, aggregatorScored) => {
+      if (typeof value === 'boolean') {
+        return value ? 'good' : 'poor';
+      }
+
+      // Limit the rating to average if this is a rating for Best Practices.
+      let rating = aggregatorScored ? 'average' : 'poor';
+      if (value > 0.33) {
```

Comment:
Agreed, and there's always a limit so we can actually make progress, PRs aren't weighted down by the past, etc. But small simplifications along the way are also a good way to help pay down technical debt. People usually need to get new stuff done so pure refactor PRs don't come along that often.